morph.js

```diff
@@ -21,15 +28,37 @@ function morph (newNode, oldNode) {
     const oldName = oldKv.name
     const oldVal = oldKv.value
 
-    if (newVal !== undefined) {
+    if (newVal === undefined) {
       removeAttribute(oldNode, oldName, oldVal)
     } else if (oldVal !== undefined || newVal !== oldVal) {
       setAttribute(oldNode, newName, newVal)
     }
   })
+}
 
-  if (newNode.nodeValue) oldNode.nodeValue = newNode.nodeValue
-  if (newNode.data) oldNode.data = newNode.data
+function copyAttrsNS (newNode, oldNode) {
+  const newAttrs = newNode._attributes
+  const oldAttrs = oldNode._attributes
+  const props = xtend(newAttrs, oldAttrs)
+
+  Object.keys(props).forEach(function (namespace) {
+    const vo = props[namespace]
+
+    Object.keys(vo).forEach(function (attrName) {
+      const newKv = newAttrs[namespace][attrName] || empty
+      const newName = `${newKv.prefix}:${attrName}`
```

Comment:
Can we use non-fancy string combination here? e.g.
```js
const newName = newKv.prefix + ':' + attrName
```
This would allow this library to continue to be used without heavy compilers like `buble` and `babel`. In a future patch we should probably also replace `const` with `var` to make it work even in the oldest browsers